.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: python
 
+sudo: false
+
 python:
   - "2.7"
   - "2.6"
+  - "3.4"
+
+addons:
+  apt:
+    packages:
+      - libgeos-c1
+      - libxml2-dev
 
 install:
-  - sudo apt-get install -y libgeos-c1
   - pip install . --use-mirrors
-  - pip install -r ./requirements-standalone.txt
-  - pip install -r ./requirements-dev.txt
-  - python ./setup.py -q install
-
-before_script:
-  - sudo add-apt-repository ppa:gcpp-kalxas/ppa-tzotsos -y
```

Comment:
We switched to new libxml to be able to validate requests here:
https://github.com/geopython/pycsw/commit/7ab7a4051afa7acf0875fc51852a6141715aed56
https://github.com/geopython/pycsw/commit/32b1dc05f4d219ba7f7a20c125b050c0056e2dc9

Has something changed with Travis? Does it use Ubuntu Trusty now?